src/app/components/ThemeProvider/themes/pidgin.ts

```diff
@@ -1,12 +1,12 @@
-import { GHOST, WHITE, BLUEJAY, POSTBOX_30 } from '../palette';
+import { GHOST, WHITE, POSTBOX, POSTBOX_30 } from '../palette';
 import latin from '../fontScripts/latin';
 import helmetFontVariants from '../fontVariants/helmet';
 import withThemeProvider from '../withThemeProvider';
 import brandSVG from '../chameleonLogos/pidgin';
 
 const pidginTheme = {
   palette: {
-    BRAND_BACKGROUND: BLUEJAY,
+    BRAND_BACKGROUND: POSTBOX,
```

Comment:
I think this must have been a "test" value left in from the PR that created these files.